README.md

```diff
@@ -54,6 +54,8 @@ $ dynalab-cli test -n <name_of_your_model>
 ```
 If the integrated test is successful, you'll see "Integrated test passed" on the prompt. You can then proceed to the next step. Otherwise, please follow the on-screen instructions to check the log and fix your code / dependencies, and repeat this step until the output is error free.
 
+If there are no errors in the log file apart from workers crashing and the integrated test fails, ensure that you have sufficient resources allocated to docker.
```

Comment:
I like that you wrote something like this. Consider breaking this up into multiple sentences so it is easier to parse. What do you think of this? Also wondering what @mazhiyi thinks.

"It is possible that your machine could lack the resources to run the deployment environment in docker. If this happens, your log file will show that workers crashed but will not include an error that references your handler.py. Uploading your model could result in fully functional behavior on our server in this scenario, even though the integrated test fails."

Reply:
I think you can add this to the overall `README.md` if you are confident about it. 

As I said we are not able to completely reproduce AWS setup and integrated test is a close proxy. Adding demo dataset as we discussed may be a workaround for this.

Reply:
Thanks both, updated!